README.md

```diff
@@ -59,24 +59,27 @@ Looking for example to use `react-flexbox-grid`? Head over to [react-flexbox-gri
 
 Advanced composition
 -------
-Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you have a `MyFormInput` component that should also act as both a `Row` and a `Col`.
+Functions for generating Row and Column classNames are exported for use in other components.  For example, suppose you're using a third party component that accepts a className.
 
 ```js
-import {getRowClassNames, getColClassNames} from 'react-flexbox-grid'
+// a component that accepts a className
+import RandomComponent from 'some-package';
```

Comment:
Hm, actually they are already exported since they're just a property one the Col & Row functions. If you think it'd be a good idea, I'll convert the `MyComponent` to a class and demonstrate using them.